package.json

```diff
@@ -32,7 +32,7 @@
     "size": "size-limit",
     "analyze": "size-limit --why",
     "generate-sass-resources": "gulp",
-    "storybook": "start-storybook -p 6006",
+    "storybook": "start-storybook -p 6006 -s ./.storybook/public",
```

Comment:
Why was this added?

Reply:
In here:
`.storybook/manager-head.html`